Content.Shared/Xenoborgs/Components/XenoborgFactoryComponent.cs

```diff
@@ -0,0 +1,74 @@
+using Content.Shared.Lathe.Prototypes;
+using Content.Shared.Research.Prototypes;
+using Content.Shared.Whitelist;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Shared.Xenoborgs.Components;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState, AutoGenerateComponentPause]
+[Access(typeof(SharedXenoborgFactorySystem))]
+public sealed partial class XenoborgFactoryComponent : Component
+{
+    [DataField, AutoNetworkedField]
+    public ProtoId<LatheRecipePrototype> Recipe;
+
+    [DataField, AutoNetworkedField]
+    public ProtoId<LatheRecipePackPrototype> BorgRecipePack;
+
+    /// <summary>
+    /// An "enable" toggle for things like interfacing with machine linking
+    /// </summary>
+    [DataField, AutoNetworkedField, ViewVariables(VVAccess.ReadWrite)]
```

Comment:
```suggestion
    [DataField, AutoNetworkedField]
```
redundant, see the info message in your IDE